SDK_versioned_docs/version-3.0.0/guides/liquidity/05-adding-liquidity.md

```diff
@@ -1,13 +1,41 @@
 ---
-id: liquidity-actions
+id: adding
 title: Adding Liquidity
 ---
 
 ## Adding Liquidity to the Position
 
-Constructing the calldata for adding liquidity uses the same function call to `addCallParameters`, which takes in a `Position` and an options field of type `AddLiquidityOptions`. Since the goal of this example is to add liquidity to an already existing position, construct `IncreaseOptions` instead of `MintOptions`. `IncreaseOptions` types are defined by the interfaces `CommonAddLiquidityOptions & IncreaseSpecificOptions`. `CommonAddLiquidityOptions` is outlined above because minting also required these parameters and the reference for `IncreaseSpecificOptions` is:
+Constructing the calldata for adding liquidity uses the same function call to `addCallParameters`, which takes in a `Position` and an options field of type `AddLiquidityOptions`. Since the goal of this example is to add liquidity to an already existing position, construct `IncreaseOptions` instead of `MintOptions`. `IncreaseOptions` types are defined by the interfaces `CommonAddLiquidityOptions & IncreaseSpecificOptions`: 
+Here is the reference in the docs for [`CommonAddLiquidityOptions`](https://docs.uniswap.org/sdk/reference/interfaces/CommonAddLiquidityOptions) and for [`IncreaseSpecificOptions`](https://docs.uniswap.org/sdk/reference/interfaces/IncreaseSpecificOptions).
```

Comment:
I would just put the links on the interface names at the end of line 8 and omit line 9.  Also omit the colon